.idea/modules.xml

```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/dartdoc.iml" filepath="$PROJECT_DIR$/dartdoc.iml" />
+      <module fileurl="file://$PROJECT_DIR$/../mustache4dart/mustache4dart.iml" filepath="$PROJECT_DIR$/../mustache4dart/mustache4dart.iml" />
```

Comment:
I would prefer to just delete the whole file so I can have my own intellij settings.  Is that a bad idea?